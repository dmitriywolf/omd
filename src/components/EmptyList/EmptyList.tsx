import React from 'react';

const EmptyList = () => {

  return (
    <div className="container">
      <h2>Use the search form to find films...</h2>
    </div>
  )
};

export default EmptyList;
