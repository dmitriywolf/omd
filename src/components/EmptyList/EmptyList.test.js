import React from 'react';
import EmptyList from './EmptyList';
import {shallow} from 'enzyme';
import toJson from "enzyme-to-json";

describe("EmptyList component:", () => {
  it("should render EmptyList component", () => {
    const component = shallow(<EmptyList/>);
    expect(toJson(component)).toMatchSnapshot();
  })
});
