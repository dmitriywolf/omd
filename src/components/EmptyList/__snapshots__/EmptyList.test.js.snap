// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyList component: should render EmptyList component 1`] = `
<div
  className="container"
>
  <h2>
    Use the search form to find films...
  </h2>
</div>
`;
